package.json

```diff
@@ -60,6 +60,7 @@
         "@types/ua-parser-js": "^0.7.33",
         "@types/uuid": "^3.4.5",
         "case-sensitive-paths-webpack-plugin": "^2.2.0",
+        "core-js-bundle": "^3.1.4",
```

Comment:
from the warning, this are **peer dependencies**, does it make sense to use `peerDependencies` property instead of `devDependencies`??

Reply:
Good question! My understanding of the warning and my reading of [how npm describes Peer Dependencies as working](https://nodejs.org/es/blog/npm/peer-dependencies/) is that these packages are peerDependencies *of the packages directly referencing them* (`typeson` and `base64-arraybuffer-es6`), but that the expectation is that this should be resolved by having the higher level package consuming `typeson`/`base64-arraybuffer-es6` have a dependency on both. Frankly, I think it would probably be appropriate for this extra dep I'm applying to instead by applied by one of the intermediate packages involved (`realistic-structured-clone` or `fake-indexeddb`), but I didn't want to spend the time driving bugs with them when the fix is very straightforward for us.

Reply:
I'm fine with completing this PR as is right now. I'm guessing we may spend some more time getting more clarity about peer dependencies at some point.